cmd/argo/commands/clustertemplate/lint.go

```diff
@@ -37,17 +36,17 @@ func NewLintCommand() *cobra.Command {
 				Strict:           strict,
 				DefaultNamespace: client.Namespace(),
 				Formatter:        fmtr,
+				Output:           os.Stdout,
 			})
 			errors.CheckError(err)
 
-			fmt.Print(res.Msg())
 			if !res.Success {
 				os.Exit(1)
 			}
 		},
 	}
 
-	command.Flags().StringVar(&format, "format", "pretty", "Linting results output format. One of: pretty|simple")
+	command.Flags().StringVarP(&output, "output", "o", "pretty", "Linting results output format. One of: pretty|simple")
```

Comment:
Why this change? Is this what we do in Argo CD or something?

Reply:
@alexec suggested that we should switch from `format` to `output` in the [last PR](https://github.com/argoproj/argo-workflows/pull/5095#issuecomment-782241555), but I didn't have time to change it before it was merged.

Reply:
SGTM